flent/runners.py

```diff
@@ -680,8 +680,17 @@ def find_binary(self, **args):
             args['socket_timeout'] = ""
 
         if args['test'] in ("TCP_STREAM", "TCP_MAERTS", "omni"):
+            self.ctrl_port = args['netperf_control_port']
```

Comment:
Huh, whatdoyaknow, so it is. It's not used, though, and so never propagated to the args dict. Guess I added the command line parameter at some point, but never got around to actually using it...